optimum/exporters/onnx/base.py

```diff
@@ -27,16 +27,12 @@
 from typing import TYPE_CHECKING, Any, Dict, Iterable, List, Optional, Tuple, Union
 
 import numpy as np
-import onnx
 from transformers.utils import is_accelerate_available, is_torch_available
 
-from ...onnx import remove_duplicate_weights_from_tied_info
-
 
 if is_torch_available():
     import torch.nn as nn
 
-from ...onnx import merge_decoders
```

Comment:
merge_decoders imported from graph_transformations submodule that contains onnx import
https://github.com/huggingface/optimum/blob/main/optimum/onnx/graph_transformations.py#L19

as this file heavily depends on  onnx functionality, while merge decoders used only for specific configs postprocessing triggered only if specific config behaviour is selected, I think it may be better to allow use other functional from this module without necessarily to use onnx import.

As I said there is case, when import onnx crashed on windows. This file contains basic functionality for configs that we also reuse for enabling new models in optimum-intel moving this import allow us continue usage of base config classes even if onnx broken